helm.tf

```diff
@@ -154,7 +154,7 @@ resource "helm_release" "metrics_server" {
 
   set {
     name = "priorityClassName"
-    value = var.metrics_autoscaler_priorityclass
+    value = var.metrics_server_priorityclass
```

Comment:
@federeynimbux i think this mismatch the name in README (it is written like metrics_server_priorityClassName)

Reply:
changes were made